src/core/agent/event.types.ts

```diff
@@ -94,6 +96,13 @@ interface IdentifierRemovedEvent extends BaseEventEmitter {
   };
 }
 
+interface IdentifierStateChangedEvent extends BaseEventEmitter {
```

Comment:
Since we removed the event emitter changes as I asked, we can revert ALL of the changes in this PR related to it. This is now unused.